crates/ironrdp-client/src/config.rs

```diff
@@ -325,6 +372,9 @@ impl Config {
                 .context("Address prompt")?
                 .pipe(Destination::new)?
         };
+        if let Some(ref mut gw) = gw {
```

Comment:
style: Spacing
```suggestion
};

        if let Some(ref mut gw) = gw {
```

Reply:
You mean just the separation spacing to the upper line/block?
    Because this also seems to move the `};` to a non-indented position. 
I also meanwhile ran cargo fmt.

Reply:
Added.